enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -51,6 +51,9 @@ const SdkContentWrapperInner = styled.div<
   --mb-color-text-dark: ${({ theme }) => theme.fn.themeColor("text-dark")};
   --mb-color-text-medium: ${({ theme }) => theme.fn.themeColor("text-medium")};
   --mb-color-text-light: ${({ theme }) => theme.fn.themeColor("text-light")};
+  --mb-color-bg-error: ${() => color("bg-error")};
+  --mb-color-bg-medium: ${({ theme }) => theme.fn.themeColor("bg-medium")};
+  --mb-color-bg-night: ${() => color("bg-night")};
```

Comment:
Should we also use `theme.fn.themeColor` here and add it to the main palette as well? (Similar to the above notes by Kelvin)